test/rules/assertions/requireParam.js

```diff
@@ -414,19 +480,27 @@ export default {
       /**
        *
        */
-      function quux ({bar, baz}, foo) {
+      function quux ({bar, baz___}, foo) {
       }
       `,
       errors: [
+        {
+          message: 'Missing JSDoc @param "root.bar" declaration.',
+        },
+        {
+          message: 'Missing JSDoc @param "root.baz___" declaration.',
+        },
         {
           message: 'Missing JSDoc @param "foo" declaration.',
         },
       ],
       output: `
       /**
+       * @param root.bar
+       * @param root.baz___
        * @param foo
        */
-      function quux ({bar, baz}, foo) {
+      function quux ({bar, baz___}, foo) {
```

Comment:
Also here...

Reply:
This was accidentally leftover from debugging, so I've removed it.